doc/menu_lua_api.txt

```diff
@@ -241,133 +241,133 @@ Package - content which is downloadable from the content db, may or may not be i
-^ look up the translation of a string in the gettext message catalog
-fgettext_ne(string, ...)
-^ call core.gettext(string), replace "$1"..."$9" with the given
-^ extra arguments and return the result
-fgettext(string, ...) -> string
-^ same as fgettext_ne(), but calls core.formspec_escape before returning result
-core.parse_json(string[, nullvalue]) -> something (possible in async calls)
-^ see core.parse_json (lua_api.txt)
-dump(obj, dumped={})
-^ Return object serialized as a string
-string:split(separator)
-^ eg. string:split("a,b", ",") == {"a","b"}
-string:trim()
-^ eg. string.trim("\n \t\tfoo bar\t ") == "foo bar"
-core.is_yes(arg) (possible in async calls)
-^ returns whether arg can be interpreted as yes
-minetest.encode_base64(string) (possible in async calls)
-^ Encodes a string in base64.
-minetest.decode_base64(string) (possible in async calls)
-^ Decodes a string encoded in base64.
+* `core.get_us_time()`
+  * returns time with microsecond precision
+* `core.gettext(string)` -> string
+  * look up the translation of a string in the gettext message catalog
+* `fgettext_ne(string, ...)`
+  * call `core.gettext(string)`, replace "$1"..."$9" with the given
+  extra arguments and return the result
+* `fgettext(string, ...)` -> string
+  * same as `fgettext_ne()`, but calls `core.formspec_escape` before returning result
+* `core.parse_json(string[, nullvalue])` -> something (possible in async calls)
+  * see `core.parse_json` (`lua_api.txt`)
+* `dump(obj, dumped={})`
+  * Return object serialized as a string
+* `string:split(separator)`
+  * eg. `string:split("a,b", ",")` == `{"a","b"}`
+* `string:trim()`
+  * eg. `string.trim("\n \t\tfoo bar\t ")` == `"foo bar"`
+* `core.is_yes(arg)` (possible in async calls)
+  * returns whether `arg` can be interpreted as yes
+* `minetest.encode_base64(string)` (possible in async calls)
+  * Encodes a string in base64.
+* `minetest.decode_base64(string)` (possible in async calls)
+  * Decodes a string encoded in base64.
 
 
 Async
 -----
 
-core.handle_async(async_job,parameters,finished)
-^ execute a function asynchronously
-^ async_job is a function receiving one parameter and returning one parameter
-^ parameters parameter table passed to async_job
-^ finished function to be called once async_job has finished
-^    the result of async_job is passed to this function
-
-Limitations of Async operations
- -No access to global lua variables, don't even try
- -Limited set of available functions
-    e.g. No access to functions modifying menu like core.start,core.close,
-    core.show_path_select_dialog
+* `core.handle_async(async_job,parameters,finished)`
+  * execute a function asynchronously
+  * `async_job` is a function receiving one parameter and returning one parameter
+  * `parameters` parameter table passed to `async_job`
+  * `finished` function to be called once `async_job` has finished
+    the result of `async_job` is passed to this function
+
+### Limitations of `Async` operations
```

Comment:
Why "`Async`" and not "Async"?

Reply:
In `lua_api.txt`, it's written Async, without anything surrounding it, so I think I'll remove the backticks.